lacasera/vue-task
# Implements Batch Updates

#### What this does
Adds implementation for running batch updates

#### How should this be manually tested?
1. run migrations and seed database
2. setup and start redis . check redis status by running `redis-server`
3. install composer dependencies
4. run `php artisan users:update` to update seeded users ( do this a few times till you get over 1000 records in `pending_updates_request` table)
5. run `php artisan batch:updates`  to sync data with third party api
6. check logs to ensure user details are logged
